test/parallel_api/numeric/numeric.ops/reduce_by_segment_zip.pass.cpp

```diff
@@ -87,14 +85,6 @@ test_with_usm(BinaryOp binary_op)
     dt_helper7.retrieve_data(output_values1);
     dt_helper8.retrieve_data(output_values2);
 
-//Dump
-#if 0
```

Comment:
You may wish to revert this change based on the previous discussion(s) in https://github.com/uxlfoundation/oneDPL/pull/2310#discussion_r2163984846. I do not have a strong preference though.